client/src/cmdusart.c

```diff
@@ -163,7 +164,12 @@ static int usart_bt_testcomm(uint32_t baudrate, uint8_t parity) {
     ret = usart_txrx((uint8_t *)string, strlen(string), data, &len, 1000);
     if (ret == PM3_SUCCESS) {
         PrintAndLogEx(SUCCESS, "RX (%3zu):%.*s", len, (int)len, data);
-        if (str_startswith((char *)data, "hc01.comV2.0") || str_startswith((char *)data, "BT SPP V3.0")) {
+        
+        // www.hc01.com and hc01.com both tells they are HC modules
+        // compatiable with HC-04 HC-05 HC-06
+        if (str_startswith((char *)data, "www.hc01.com") ||
+        str_startswith((char *)data,"hc01.com") ||
+        str_startswith((char *)data, "BT SPP 4V3.0")) {
```

Comment:
that `4`  will break the previous detection

Reply:
I'm not sure what you mean, is `BT SPP 4V3.0`? I think this will work with other modules so I kept it, but I don't know which module replies this because it's in the original code.

Reply:
I rewrote it in new commit, and edited this typo.